features/collections/collection_notification.feature

```diff
@@ -47,6 +47,18 @@ Feature: Collectible items email
     And I should see "Antarctic Penguins"
     And 1 email should be delivered to test2@archiveofourown.org
 
+  Scenario: Work added to collection by Archivist sends notification email
+    Given I have a work "cool work"
+      And I have an archivist "archivist"
+      And all emails have been delivered
+    When I am logged in as "archivist"
+      And I create the collection "Cold Collection" with name "ColdCollection"
+      And I view the work "cool work"
+      And I follow "Add to Collections"
+      And I fill in "collection_names" with "ColdCollection"
+      And I press "Add"
+    Then 1 email should be delivered
```

Comment:
Could you add a check for the email subject or text to make sure it's the right email?

Reply:
FYI looks like #5023 is adding the same test for this mailer. Let's get that one merged first.

Reply:
I've removed the test (as it is slightly tangential to the bug being fixed). If necessary, I can append a check for the email recipient and body to the test in #5023 after it is merged.